tests/context.py

```diff
@@ -0,0 +1,77 @@
+import os
+from pathlib import Path
+
+import jpype
+import jpype.imports
+from jpype.types import *
+
+REPO_DIR = Path(__file__).parent.parent
+schema_file = Path.joinpath(REPO_DIR, "tests/data/tpch_parquet/schema.sql")
+
+calcite_jars = Path.joinpath(REPO_DIR, "jars/*")
```

Comment:
I would encapsulate these global definitions (lines 11-24) into a separate python file/module.  So, in the future we can change -if we want/need to- the integration layer we are using to another thing that is not jpype.

Reply:
Yes, I agree with @aucahuasi. Also we could use config files like `yml` to dynamically configure such attributes. If they are experiment bound, the user just have to do 
`consumer-testing run --config some_yaml.yaml` 

Just a thought.